release_notes/46-release-notes.adoc

```diff
@@ -440,4 +441,17 @@ This release of {product-title-short} fixes the following bugs:
 
 * Fixed an issue where the output format of the `roxctl` CLI command for timestamps changed in {product-title-short} 4.6, interrupting customer automation workflows.
 
+[id="about-release-463_{context}"]
+== About release 4.6.3
+
+*Release date*: 10 March 2025
+
+This release of {product-title-short} fixes the following bugs:
+
+//ROX-26908
+* Fixed an issue where the Operator was not displayed as Federal Information Processing Standards (FIPS) compliant in the Operator catalog search.
+
+//ROX-27347
+* Fixed an issue where vulnerability report jobs in {product-title-short} could become deadlocked if a report took too long to complete, preventing new jobs from starting.
```

Comment:
The same set applies here
- https://github.com/openshift/openshift-docs/pull/89535/files#r1985041862

---

PRs:
- https://github.com/stackrox/stackrox/pull/14505
- https://github.com/stackrox/stackrox/pull/14433
- https://github.com/stackrox/stackrox/pull/14467